apps/cms/test/paragraph_test.exs

```diff
@@ -293,7 +293,7 @@ defmodule CMS.ParagraphTest do
     end
 
     test "parses an agenda topic paragraph" do
-      assert [topic_1, topic_2] = api_paragraph("agenda_topic")
+      assert [topic_1, _topic_2] = api_paragraph("agenda_topic")
```

Comment:
Oo, are there new linting warnings about unused vars here? That would be neat.

Reply:
Yeah, thanks Elixir 1.11!